src/nix/hash.cc

```diff
@@ -19,47 +20,12 @@ struct CmdHashBase : Command
     FileIngestionMethod mode;
     HashFormat hashFormat = HashFormat::SRI;
     bool truncate = false;
-    HashAlgorithm ha = HashAlgorithm::SHA256;
+    HashAlgorithm hashAlgo = HashAlgorithm::SHA256;
     std::vector<std::string> paths;
     std::optional<std::string> modulus;
 
     explicit CmdHashBase(FileIngestionMethod mode) : mode(mode)
     {
-        addFlag({
-            .longName = "sri",
-            .description = "Print the hash in SRI format.",
-            .handler = {&hashFormat, HashFormat::SRI},
-        });
-
-        addFlag({
-            .longName = "base64",
-            .description = "Print the hash in base-64 format.",
-            .handler = {&hashFormat, HashFormat::Base64},
-        });
-
-        addFlag({
-            .longName = "base32",
-            .description = "Print the hash in base-32 (Nix-specific) format.",
-            .handler = {&hashFormat, HashFormat::Nix32},
-        });
-
-        addFlag({
-            .longName = "base16",
-            .description = "Print the hash in base-16 format.",
-            .handler = {&hashFormat, HashFormat::Base16},
-        });
-
-        addFlag(Flag::mkHashAlgoFlag("type", &ha));
-
-        #if 0
-        addFlag({
-            .longName = "modulo",
-            .description = "Compute the hash modulo the specified string.",
-            .labels = {"modulus"},
-            .handler = {&modulus},
-        });
-        #endif\
```

Comment:
We ought to support this; left commented out but moved to `nix hash path` since we don't need to add new features to deprecated commands.